Dockerfile

```
@@ -24,7 +24,8 @@ RUN git config --global url."https://".insteadOf git://
 
 # Install our node/python requirements
 RUN npm install -g npm@6.14.5
```

```suggestion
RUN nvm use
```
We should update `.nvmrc` to the current version, and then this command would read from that and install+use that version.

---

It now crashes because it can't find `nvm`; I'm guessing we also need to install `nvm` or something?

---

Guess so

---

I've just removed NVM as I don't think we need to worry about handling multiple node versions in the context of docker deploys or CI.